<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelAxis">
        <property name="text">
         <string>Default Axis</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelIndex">
        <property name="text">
         <string>Matrix Indexing Mode</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelTheme">
        <property name="text">
         <string>Theme</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBoxTheme"/>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="checkBoxAxis">
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="checkBoxIndex">
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxIndex</sender>
   <signal>toggled(bool)</signal>
   <receiver>Dialog</receiver>
   <slot>updateIndexCheckboxLabel(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxAxis</sender>
   <signal>toggled(bool)</signal>
   <receiver>Dialog</receiver>
   <slot>updateAxisCheckboxLabel(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>392</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>applyPreferences()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>66</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateAxisCheckboxLabel(bool)</slot>
  <slot>updateIndexCheckboxLabel(bool)</slot>
  <slot>applyPreferences()</slot>
 </slots>
</ui>
